src/EditorFeatures/CSharp/AutomaticCompletion/Sessions/LessAndGreaterThanCompletionSession.cs

```diff
@@ -35,6 +35,7 @@ public override bool CheckOpeningPoint(IBraceCompletionSession session, Cancella
             // type argument or parameter list
             if (!token.CheckParent<TypeParameterListSyntax>(n => n.LessThanToken == token) &&
                 !token.CheckParent<TypeArgumentListSyntax>(n => n.LessThanToken == token) &&
+                !token.CheckParent<FunctionPointerTypeSyntax>(n => n.LessThanToken == token) &&
```

Comment:
is there test for this?

Reply:
Added `FunctionPointerStartSession` in `AutomaticLessAndGreaterThanCompletionTests`